projects/plugins/jetpack/_inc/client/main.jsx

```diff
@@ -311,7 +311,9 @@ class Main extends React.Component {
 
 	shouldShowFooter() {
 		// Only show on the dashboard and settings page
```

Comment:
Update the comment too?
```suggestion
		// Only show on the dashboard, settings, and recommendations pages
```

Reply:
Good point. I committed your suggested changes.